test/parallel/test-runner-output.mjs

```diff
@@ -15,7 +15,7 @@ function replaceTestDuration(str) {
 }
 
 const color = '(\\[\\d+m)';
-const stackTraceBasePath = new RegExp(`${color}\\(${process.cwd()}/?${color}(.*)${color}\\)`, 'g');
+const stackTraceBasePath = new RegExp(`${color}\\(${process.cwd().replace(/[\\^$*+?.()|[\]{}]/g, '\\$&')}/?${color}(.*)${color}\\)`, 'g');
```

Comment:
```suggestion
const stackTraceBasePath = new RegExp(`${color}\\(${RegExpPrototypeSymbolReplace(/[\\^$*+?.()|[\]{}]/g, process.cwd(), '\\$&')}/?${color}(.*)${color}\\)`, 'g');
```

Reply:
I missed that this is a test file, where we control the environment, usage of primordials is unnecessary. Feel free to revert my suggestion.